hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/protocolPB/OzoneManagerProtocolServerSideTranslatorPB.java

```diff
@@ -193,6 +193,15 @@ private OMResponse submitReadRequestToOM(OMRequest request)
     }
   }
 
+  private ServiceException createLeaderErrorException(
```

Comment:
Minor: Unnecessary code change. I see it is just moved up.

Reply:
Yes functionally no change. Moved it up for aesthetics and improving code readability.

Reply:
I am fine with it, but one thing with this is we will lose which commit originally added this. (As sometimes I check why this code is added/changed to understand the reason for this code change. This is just my personal opinion.)